user_list.php

```diff
@@ -29,64 +29,46 @@
 			<!-- import the webpage's javascript -->
 			<!-- nav javascript -->
 			<script src="js/nav.js" defer></script>
-			<!-- back to top button javascript -->
-			<script src="js/backtotop.js" defer></script>
 		</head>
 	
         <body>
-              <!-- logo at top of page-->
-			<center><a href="index.php"><img style="width: 100%; max-width: 100px;" src="images/placeholder.png" alt="TuneIn logo"></a></center>	
-
 			<!-- navigation menu (burger menu)-->
 			<?php
 			  require("nav.php");
 			?>
 			
-                                <div class="aside"><!-- holds the admin menu with links to the different pages only the admin can access -->
-									<ul>
-										<li><a href="user_list.php">User List</a></li> 
-										<li><a href="add_user.php">Add New User</a></li>
-										<li><a href="update_user.php">Update User</a></li>
-										<li><a href="delete_user.php">Delete User</a></li>
-									</ul>
-								</div>
-			
-			
-                                <div class="admin_content"><!-- Holds the main page content -->
-								<h2>USERS</h2>
-								<?php
-									//connect.php (tells where to connect servername, username, password, dbaseName)
-									require "3.3_Assessment_FTuason_mysqli.php";
+                                <div class="content" style="padding-top: 0;"><!-- holds the main page content -->
```

Comment:
#instead of having a separate content class for my admin pages I used the same styling for all main content